Fix Pancheros name

aenet14 opened the pull request:
Removed apostrophes from displayname and tags brand and name for Pancheros Mexican Grill to match wikidata and official Pancheros website

Snowysauce commented:
Thanks for noticing the error. I corrected the name on Wikidata while evaluating the PR that added the brand to the NSI, but I completely failed to notice that the brand had the wrong values in the PR itself. The next NSI release isn't scheduled for a week, so hopefully there won't be too much damage to the OSM data in the interim.

aenet14 commented:
I noticed it while I was correcting warnings on OSM. There’s a Pancheros
location near me that it said was spelled incorrectly, but Mapillary showed
it without the apostrophe, which led to me to check their website and the
wikidata, which both lined up with not having an apostrophe. So instead of
changing the location, I went I changed the NSI instead.

On Sun, Feb 16, 2025 at 3:06 PM Snowysauce ***@***.***> wrote:

> Thanks for noticing the error. I corrected the name on Wikidata while
> evaluating the PR that added the brand to the NSI, but I completely failed
> to notice that the brand had the wrong values in the PR itself. The next
> NSI release isn't scheduled for a week, so hopefully there won't be too
> much damage to the OSM data in the interim.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/osmlab/name-suggestion-index/pull/10531#issuecomment-2661594793>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO4VB6IC5DPXL5ETUYDWLYL2QDVVFAVCNFSM6AAAAABXHYSTVCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNRRGU4TINZZGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: Snowysauce]*Snowysauce* left a comment
> (osmlab/name-suggestion-index#10531)
> <https://github.com/osmlab/name-suggestion-index/pull/10531#issuecomment-2661594793>
>
> Thanks for noticing the error. I corrected the name on Wikidata while
> evaluating the PR that added the brand to the NSI, but I completely failed
> to notice that the brand had the wrong values in the PR itself. The next
> NSI release isn't scheduled for a week, so hopefully there won't be too
> much damage to the OSM data in the interim.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/osmlab/name-suggestion-index/pull/10531#issuecomment-2661594793>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO4VB6IC5DPXL5ETUYDWLYL2QDVVFAVCNFSM6AAAAABXHYSTVCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNRRGU4TINZZGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
